src/category_theory/abelian/basic.lean

```diff
@@ -81,20 +81,24 @@ variables (C)
 section prio
 set_option default_priority 100
 
-/-- A (preadditive) category `C` is called abelian if it has a zero object, all binary products and
-    coproducts, all kernels and cokernels, and if every monomorphism is the kernel of some morphism
+/-- A (preadditive) category `C` is called abelian if it has all finite products,
+    all kernels and cokernels, and if every monomorphism is the kernel of some morphism
     and every epimorphism is the cokernel of some morphism. -/
 class abelian extends preadditive.{v} C :=
-[has_zero_object : has_zero_object.{v} C]
```

Comment:
I'm probably missing something here, but why are we guaranteed to have a zero object now? Finite products gives an terminal object but why does it coincide with the initial object?

Reply:
Oh sorry this is addressed below, ignore